crates/goose-cli/src/commands/configure.rs

```diff
@@ -441,6 +442,8 @@ pub fn configure_extensions_dialog() -> Result<(), Box<dyn Error>> {
                 enabled: true,
                 config: ExtensionConfig::Builtin {
                     name: extension.clone(),
+                    // NOTE: default timeout for builtin tools
```

Comment:
nit: probably useful to ask for a timeout option here as well! i think a few CLI users have expressed an interest increasing this for the developer extension